MANIFEST.in

```diff
@@ -4,6 +4,7 @@ include *.rst
 include *.txt
 include buildout.cfg
 include tox.ini
+include appveyor.yml
```

Comment:
and now we have both "include appveyor.yml" and "exclude appveyor.yml", this metaconfig stuff is ... not ideal.

Reply:
fixed.